scripts/wsl2_ai_tuning.py

```
@@ -0,0 +1,401 @@
+        except Exception as e:
+            print(f"  [-] Failed writing /etc/wsl.conf: {e}")
+
+    # 4. .wslconfig persistence & User Profile discovery
+    wslconfig_text = generate_wslconfig(wsl_mem_gb, wsl_cpus, networking_mode)
+    persist_dir = Path("/opt/dsom-persistence")
+    if os.geteuid() == 0:
+        try:
+            persist_dir.mkdir(parents=True, exist_ok=True)
+            (persist_dir / "wsl2_tuned.wslconfig").write_text(wslconfig_text)
+            print(f"  [+] Tuned .wslconfig saved to {persist_dir}/wsl2_tuned.wslconfig")
+        except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)